metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1486,13 +1488,27 @@ def extract_independent_looks(self) -> Iterable[MetadataWorkUnit]:
         )
         for look in all_looks:
             if look.id in self.reachable_look_registry:
-                # This look is reachable from Dashboard
+                # This look is reachable from the Dashboard
                 continue
 
             if look.query_id is None:
                 logger.info(f"query_id is None for look {look.title}({look.id})")
                 continue
 
+            if self.source_config.skip_personal_folders:
+                if look.folder is not None and (
+                    look.folder.is_personal or look.folder.is_personal_descendant
+                ):
+                    self.reporter.info(
+                        title="Dropped Look",
+                        message="Dropped due to being a personal folder",
+                        context=f"Look ID: {look.id}",
+                    )
```

Comment:
does this reporting match what we do for other dashboards/looks in personal folders?

Reply:
yes